docs/reference/api/storage/LittleFileSystem.md

```diff
@@ -1,11 +1,60 @@
 ## LittleFileSystem
 
-[Add description here.]
+The little filesystem (LittleFS) is a fail-safe filesystem designed for embedded systems, specifically for microcontrollers that use flash storage.
+
+```
+   | | |     .---._____
+  .-----.   |          |
+--|o    |---| LittleFS |
+--|     |---|          |
+  '-----'   '----------'
+   | | |
+```
+
+Microcontrollers and flash storage present three challenges for embedded storage: [power loss](#power-loss-resilience), [wear](#wear-leveling) and [limited RAM and ROM](#bounded-ram-and-rom). The LittleFS provides a solution to all three of these problems.
+
+- **Bounded RAM/ROM** - The LittleFS works with a limited amount of memory. The LittleFS avoids recursion and limits dynamic memory to configurable buffers that can be provided statically.
+
+- **Power-loss resilient** - The LittleFS is designed for operating systems that may have random power failures. The LittleFS has strong copy-on-write guarantees and keeps storage on disk in a valid state.
+
+- **Wear leveling** - Because the most common form of embedded storage is erodible flash memories, the LittleFS provides a form of dynamic wear leveling for systems that cannot fit a full flash translation layer.
+
+### When to use the LittleFS
+
+The LittleFS is intended for microcontrollers with external flash storage. In this context, the LittleFS outperforms the other Mbed OS file systems in terms of RAM, ROM, wear and runtime.
+
+For storage on an SD card that is accessible from a PC, use the FATFileSystem due to its portability.
```

Comment:
if portability is required between other operating systems and fail-safe operation isn't mandatory, the FATFileSystem would be a better choice

Reply:
Is this not more or less what is said here? At least that's how I read it.

Reply:
@sg- How would you like us to clarify?